core/templatecache.go

```diff
@@ -21,7 +23,7 @@ func (tc *TemplateCache) Load(text string) (*template.Template, error) {
 		return cachedTemplate, nil
 	}
 
-	templ, err := template.New(text).Parse(text)
+	templ, err := template.New(text).Funcs(sprig.TxtFuncMap()).Parse(text)
```

Comment:
Mind adding a test for a simple sprig function in a template, just as a sanity check.

Reply:
Added